### PR TITLE
Working on the Vault Group (Stake Page)

### PR DESCRIPTION
**Pull request by nora360**

### CODE REVIEW

File: frontend/src/pages/spotnet/Vault/Section.jsx
Comment (whateverfw) [inline frontend/src/pages/spotnet/Vault/Section.jsx:1]: it seems like this component is not used and not needed at all

File: frontend/src/pages/spotnet/Vault/TableOfContents.jsx
Comment (whateverfw) [inline frontend/src/pages/spotnet/Vault/TableOfContents.jsx:1]: this is an exact copy of the component from a documentation page, can you instead move it to components folder and use the same component for documentation and here (passing correct items as props)

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
@nora360 Please, finish your PR and fix comments above

**Comment by whateverfw** [comment]
@nora360 also, be sure to sync with main and resolve merge conflicts

**Comment by djeck1432** [comment]
@nora360 You were unassigned

**Pull request closed without merging**